src/components/Disputes/hooks/useDisputeSubscription.js

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react'
+import { useSubscription } from 'urql'
+import { reduceDispute } from '../reducer'
+import { SingleDispute } from '../../../queries/disputes'
+
+export default function useSingleDisputeSubscription(id) {
```

Comment:
Should we use `useQuery` instead of useSubscription if we are directly querying by id?

Reply:
I was thinking that a dispute might update its data, while we are on the page?

Reply:
Yes, we will need a subscription here.

Reply:
yeap totally true